standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -894,19 +894,25 @@ private void updateReplId(Connection dbConn, ReplLastIdInfo replLastIdInfo) thro
               quoteString(db), quoteString(catalog));
       rs = pst.executeQuery();
       if (rs == null || !rs.next()) {
-        throw new MetaException(" db with name " + db + " does not exist in catalog " + catalog);
+        throw new MetaException("DB with name " + db + " does not exist in catalog " + catalog);
       }
       long dbId = rs.getLong(1);
 
       if (needUpdateDBReplId) {
         rs = stmt.executeQuery("select PARAM_VALUE from DATABASE_PARAMS where PARAM_KEY = " +
```

Comment:
Do we need to append "FOR UPDATE" as we are going to update the row if exist? If yes, need change for TABLE and PARTITION PARAMS too.

Reply:
as its only single thread which updates this for the table in replication flow ..for update is not required